finality-aleph/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache 2.0"
 aleph-bft = "0.18.1"
 aleph-bft-rmc = "0.4.0"
 aleph-primitives = { package = "primitives", path = "../primitives" }
-aleph-aggregator = { package = "aggregator", path = "../aggregator" }
+aleph-aggregator = { package = "aggregator", git = "https://github.com/Cardinal-Cryptography/aleph-node.git", tag = "aggregator-rmcv0.4" }
```

Comment:
Uh, the tag should probably be versioned by the version of aggregator itself, rather than its dependencies.

Reply:
will rename

Reply:
done